fallzahlen_kanton_total_csv/COVID19_Fallzahlen_Kanton_JU_total.csv

```diff
@@ -1,6 +1,22 @@
 date,time,abbreviation_canton_and_fl,ncumul_tested,ncumul_conf,ncumul_hosp,ncumul_ICU,ncumul_vent,ncumul_released,ncumul_deceased,source
-2020-03-19,16:00,JU,"",27,6,"","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Accueil/Coronavirus-Informations-officielles-a-la-population-jurassienne.html
-2020-03-20,17:00,JU,"",29,5,"","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Accueil/Coronavirus-Informations-officielles-a-la-population-jurassienne.html
+2020-03-03,"",JU,"",1,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-04,"",JU,"",1,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-05,"",JU,"",2,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-06,"",JU,"",3,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-07,"",JU,"",4,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-08,"",JU,"",4,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-09,"",JU,"",5,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-10,"",JU,"",6,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-11,"",JU,"",6,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
```

Comment:
Jura reports 0 for three dates (4.3, 8,3, 11.3). This is a bit weird, as this makes no sense when reporting cumulative figures. The value for the 25.3 is very close to that of the BAG, so it truly seems to be a cumulative number and not something else (like the  number of currently positive). Just to ensure everything is fine here we could ask, what these 0s mean?

Reply:
Yes, good question. Looking at the number it looks like the zero means "no reporting today", because this would almost perfectly fit to the models.

Reply:
I've just mailed them to ask.

Reply:
Answer:

Monsieur,

Il s’agit du total des cas confirmés testés positifs au test du coronavirus.


Cordialement